scripts/prepare_csv.py

```diff
@@ -0,0 +1,182 @@
+import json
+import logging
+import sys
+import warnings
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+from tqdm import tqdm
+
+warnings.filterwarnings("ignore")
+
+import torch
+from torch.utils.data import random_split
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+logger = logging.getLogger(__name__)
+sys.path.append(str(wd))
+
+from lit_gpt.tokenizer import Tokenizer
+
+SEED = 42
+MASK_INPUTS = False
+IGNORE_INDEX = -1
+TEST_SPLIT_FRACTION = 0.1
+CHECKPOINT_DIR = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
+CSV_PATH = "instruction.csv"
```

Comment:
this should be user provided value

Reply:
Well, I was taking inspiration from the other files, hence I wrote the code in this way.